tests/ReactHttpAdapterTest.php

```diff
@@ -48,6 +48,18 @@ public function testSendWithTimeoutExceeded($timeout)
         $this->markTestSkipped();
     }
 
+    public function testSendWithSelfSignedSslCertificate()
```

Comment:
Note to myself, this test shouldn't be skipped but for a reason I don't understand for now, React fails for PHP >= 5.6 only.